src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -79,11 +79,15 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
         public async Task<bool> PushItemsToFeedAsync(IEnumerable<string> items, bool allowOverwrite)
         {
             Log.LogMessage(MessageImportance.Low, $"START pushing items to feed");
-            Random rnd = new Random();
 
-            try
+            if (!items.Any())
             {
+                Log.LogError("No items to push found in the items list.");
```

Comment:
This should not be an error. Just make it a message.

In a case where someone only pushes symbol.nupkg items this would be empty and we don't want that to be an error.